presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -1012,6 +1012,9 @@ public void testCreateTableAsColumns()
         assertFails(COLUMN_NAME_NOT_SPECIFIED, "CREATE TABLE test AS SELECT 123");
         assertFails(DUPLICATE_COLUMN_NAME, "CREATE TABLE test AS SELECT 1 a, 2 a");
         assertFails(COLUMN_TYPE_UNKNOWN, "CREATE TABLE test AS SELECT null a");
+        assertFails(MISMATCHED_COLUMN_ALIASES, "CREATE TABLE test(x) AS SELECT 1, 2");
+        assertFails(MISMATCHED_COLUMN_ALIASES, "CREATE TABLE test(x, y) AS SELECT 1");
+        assertFails(MISMATCHED_COLUMN_ALIASES, "CREATE TABLE test(x, y) AS (VALUES 1)");
```

Comment:
Added simple analyze tests. It does not has assertion just like other methods, though.